streams/src/test/java/org/apache/kafka/streams/systemtest/ShutdownDeadlockTest.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.smoketest;
```

Comment:
I see. In that case could we just move to `org.apache.kafka.streams.tests`? The rationale is that we do not have a `systemtest` sub-package in other kafka modules yet and all `kafkatest/services` (i.e. the ducktape system tests repo) either trigger some script to start the service in-directly or depend on its mock in `o.a.k.[package].tests`.